pkgs/development/python-modules/datalad-next/default.nix

```diff
@@ -0,0 +1,106 @@
+{
+  annexremote,
+  buildPythonPackage,
+  datalad,
+  datasalad,
+  fetchFromGitHub,
+  git,
+  git-annex,
+  humanize,
+  lib,
+  more-itertools,
+  psutil,
+  pytestCheckHook,
+  setuptools,
+  openssh,
+  unzip,
+  webdavclient3,
+}:
+
+buildPythonPackage rec {
+  pname = "datalad-next";
+  version = "1.5.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "datalad";
+    repo = "datalad-next";
+    rev = "refs/tags/${version}";
+    hash = "sha256-fqP6nG2ncDRg48kvlsmPjNBOzfQp9+7wTcGvsYVrRzA=";
+  };
+
+  nativeBuildInputs = [ git ];
+
+  build-system = [ setuptools ];
```

Comment:
As I remember, we discuss change `versioneer` in `datalad-next`

Reply:
right. Good catch. Done